twilio_server.rb

```diff
@@ -0,0 +1,18 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'sinatra'
+
+
+get '/sms-quickstart' do
+  body = params['Body']
+  puts params
+  p params.inspect
+  twiml = Twilio::TwiML::Response.new do |r|
+    if body == 'burger'
+      r.Message 'buuuuurger'
+    else
+      r.Message "Thank you for hitting up Deon Dumplings, your favourite double Ds. Text ordering is not yet supported, watch this space!"
```

Comment:
Metrics/LineLength: Line is too long. [138/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)